src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -662,6 +662,17 @@ protected function compileElseif($expression)
     }
 
     /**
+     * Compile the has statements into valid PHP.
+     *
+     * @param  string  $expression
+     * @return string
+     */
+    protected function compileHas($expression)
+    {
+        return "<?php if(!empty(trim(\$__env->yieldContent{$expression}))): ?>";
```

Comment:
So should this be changed? To me this seems correct. I want to see whether there is any content to be yielded.